snuba/datasets/entities/audit_log.py

```diff
@@ -0,0 +1,45 @@
+from abc import ABC
+from typing import Optional, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime, String
+from snuba.datasets.entities.entity_data_model import EntityColumnSet
+from snuba.datasets.entity import Entity
+from snuba.datasets.plans.single_storage import SingleStorageQueryPlanBuilder
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.factory import get_writable_storage
+from snuba.pipeline.simple_pipeline import SimplePipelineBuilder
+from snuba.query.processors import QueryProcessor
+
+profile_columns = EntityColumnSet(
+    [
+        Column("event_id", UUID()),
+        Column("timestamp", DateTime()),
+        Column("event_type", String()),
+        Column("user", String()),
+        Column("details", String()),
+    ]
+)
+
+
+class AuditLogEntity(Entity, ABC):
```

Comment:
I don't think this one needs to be an abstract base class, unless you planning to subclass it?